lib/template.html

```diff
@@ -206,28 +206,60 @@
     "headline": {{ seo_page_title | jsonify }},
 {% endif %}
 
+{% if seo_author %}
+    "author": {
+      "@type": "Person",
+      "name": {{ seo_author | jsonify }}
+    },
+{% endif %}
+
 {% if seo_page_image %}
-    "image": {{ seo_page_image | jsonify }},
+    {% if page.image.height && page.image.width %}
+      "image": {
+          "@type": "ImageObject",
+          "url": {{ seo_page_image | jsonify }},
+          "height": {{ page.image.height | jsonify }},
+          "width": {{ page.image.width | jsonify }}
+      },
+    {% else %}
+        "image": {{ seo_page_image | jsonify }},
+    {% endif %}
 {% endif %}
 
 {% if page.date %}
     "datePublished": {{ page.date | date_to_xmlschema | jsonify }},
 {% endif %}
 
+{% if page.dateModified %}
+    "dateModified": {{ page.dateModified | date_to_xmlschema | jsonify }},
+{% elsif page.date %}
+    "dateModified": {{ page.date | date_to_xmlschema | jsonify }},
+{% endif %}
```

Comment:
Can you link to some documentation explaining how this information is used? Is there value in providing the modified date when it doesn't differ from the publishing date?